README.md

```diff
@@ -1,68 +1,132 @@
+    </picture>
+  </a>
+</p>
+
+[Website](https://uploadcare.com?ref=github-readme) • [Getting Started](https://uploadcare.com/docs/start/quickstart?ref=github-readme) • [Docs](https://uploadcare.com/docs?ref=github-readme) • [Blog](https://uploadcare.com/blog?ref=github-readme) • [Discord](https://discord.gg/mKWRgRsVz8?ref=github-readme) • [Twitter](https://twitter.com/Uploadcare?ref=github-readme)
+
 [![npm version](https://badge.fury.io/js/@uploadcare%2Fblocks.svg)](https://www.npmjs.com/package/@uploadcare/blocks)
+[![GitHub Actions](https://github.com/uploadcare/blocks/workflows/checks/badge.svg)](https://github.com/cube-js/cube/actions?query=workflow%3ABuild+branch%3Amaster)
+[![License](https://img.shields.io/badge/License-MIT-blue.svg)](https://opensource.org/licenses/MIT)
+
+# 📦  Uploadcare Blocks: Stack-Agnostic Library for Uploading, Processing, and Editing Images On-the-Fly
+
+`Uploadcare Blocks` is a powerful JavaScript library for creating custom file-handling services. `Blocks` include various interactive and customizable UI components to enhance users' file uploading and processing experience. As a bonus, you receive all the [Uploadcare's](https://uploadcare.com/) versatile file-handling capabilities, including [smart CDN](https://uploadcare.com/docs/delivery/cdn/#content-delivery-network), [MIME-type filtering](https://uploadcare.com/docs/moderation/#file-types), [signed uploads](https://uploadcare.com/docs/security/secure-uploads/), and [even more]((https://uploadcare.com/features/)). 
+
+<img alt="Uploadcare Blocks examples" src="https://ucarecdn.com/8035cdc7-f0b9-4ea9-8c15-05816f315481/">
+
+## Core Features
+
+- [File Uploader](/solutions/file-uploader/) — implement file managing functionality with [multifunctional or minimal interfaces](https://uploadcare.com/docs/file-uploader/) in just a few minutes.
+- [Advanced Customization](/blocks/) — create unique file-uploading interfaces by reorganizing pre-built components and styles with custom based on your needs.
+- [Adaptive Images](/solutions/adaptive-image/) — build responsive user interfaces that adapt to various screen sizes and devices, ensuring a performant experience across platforms. 
+- [Cloud Image Editor](/solutions/cloud-image-editor/) — edit, transform, and process images right from a browser on-the-fly.
+- [Uploadcare power and features](https://uploadcare.com/features/) — upload, process, deliver, and manage images without building custom infrastructure.
+
+
+- `Coming Soon!` Web Components for any media content management like audio, video, image galleries, and more.
+
+## Why Blocks?
+**Rapid Integration**
+
+`Uploadcare Blocks` provide a ready-to-use set of UI components for file handling. It saves you valuable development time and effort, allowing you to focus on other core aspects of your application. You even don't need to build a stage while developing — just run the code directly from the browser or Node.js.
```

Comment:
> just run the code directly from the browser or Node.js.

There was a mistake in the original text. Blocks code can't run under node.js.

Reply:
Removed the mention of Node.js